lib/taskana-core-test/src/test/java/acceptance/ArchitectureTest.java

```diff
@@ -602,8 +611,7 @@ public void check(JavaClass javaClass, ConditionEvents events) {
         for (JavaMethod method : javaClass.getAllMethods()) {
           List<String> sqlStrings = getSqlStringsFromMethod.apply(method);
 
-          if (sqlStrings.isEmpty()
-              && method.tryGetAnnotationOfType(SelectProvider.class).isEmpty()) {
+          if (sqlStrings.isEmpty()) {
             String message =
                 String.format(
                     "Method '%s#%s' does not contain any MyBatis SQL annotation",
```

Comment:
I remember the struggle we had with this error message. Should we rename it to something like 
"Could not extract SQL Statement from '%s#%s'. Maybe an unsupported MyBatis SQL annotation is used or no annotation is present?"

Reply:
done